tests/basic/overlay/Makefile

```diff
@@ -0,0 +1,40 @@
+include ../../common.mk
+
+PROG=/bin/touch /foo.txt
+IMAGE_SIZE=5M
+
+EXECUTION_TIMEOUT=60
+
+SGXLKL_ENV=SGXLKL_VERBOSE=1 SGXLKL_KERNEL_VERBOSE=1 SGXLKL_HD_RO=1
+SGXLKL_HW_PARAMS=--hw-debug
+SGXLKL_SW_PARAMS=--sw-debug
+
+SGXLKL_ROOTFS=sgx-lkl-rootfs.img
+
+.DELETE_ON_ERROR:
+.PHONY: all clean
+
+$(SGXLKL_ROOTFS):
+	${SGXLKL_DISK_TOOL} create --size=${IMAGE_SIZE} --docker=./Dockerfile ${SGXLKL_ROOTFS}
```

Comment:
Don't you want to be testing this with dm-verity turned on?

Reply:
I could, though there's no difference since SGXLKL_HD_RO=1 is set which achieves the same effect.

Reply:
Hmm, I'm not sure if this is indeed entirely equivalent. I think that it would be good to test this with an actual integrity protect device-mapper file system, especially given the upcoming changes to the encrypted file system handling...

Reply:
My issue is that I don't want to create a test that tests too many things at once. And adding dm-crypt + dm-verity seems like a lot if we can get away with just mounting the root disk read-only. I checked that the test fails if readonly=true and overlay=false.

Reply:
Fair enough. So we should still have another test that checks if overlayfs works with an integrity-protected file system.

Reply:
I won't add a dm-verity test because that's still badly broken. But I can do dm-crypt + dm-integrity and set the disk as readonly. OK?

Reply:
I take that back... I can do dm-crypt. Getting "bad count while changing owner" left and right with dm-integrity.